src/main/java/com/epam/lowcost/DAO/UserDAO.java

```diff
@@ -9,6 +9,8 @@
 
     User getById(long userId);
 
+    User findByEmail(String log, String pass);
```

Comment:
You find by email, ok. But why `log`, `pass` ?

Reply:
Changed onto appropriative signature.